classes/class-wc-connect-api-client.php

```diff
@@ -291,13 +291,14 @@ protected function request( $method, $path, $body = array() ) {
 				return $headers;
 			}
 
+			set_time_limit( 0 ); // Disable PHP timeout (30 seconds default)
 			$args = array(
 				'headers' => $headers,
 				'method' => $method,
 				'body' => $body,
 				'redirection' => 0,
 				'compress' => true,
-				'timeout' => 20,
+				'timeout' => 60,
```

Comment:
The `set_time_limit()` option only affects the current request. What happened is that I put the "request timeout" to 60 seconds, but since the default PHP timeout (in my machine) was 30 seconds, the process was killed anyways, so the `set_time_limit()` call was needed. Do you think it would be better if I put an explicit PHP timeout, like `set_time_limit(60)` instead of infinite?

This change wasn't needed for this PR, but testing those changes I run into that issue (maybe EasyPost was having a bad day, or maybe the "Express" USPS API is slower than the regular one). I can put it in a separate PR if you want.